### DISCUSSION TITLE
Partitioning best practices/recommendations

### DISCUSSION THREAD

**Discussion by bovy89** [rancher/rke2#1261] [Q&A]
Hi, any suggestion on what kind of partitioning strategy best fit for an RKE2 deployment?
Now I’m following recommendations from RedHat 8 CIS benchmark and I was wondering if there are official/unofficial guidelines for RKE2, for example:
- `/var/lib/rancher/rke2/server/logs` (audit logs) on a dedicated partition and at least XX GB
- `/var/lib/rancher` on a dedicated partition and at least XX GB
- `/var/lib/rancher/rke2/server/db` on a dedicated partition and at least XX GB
- `/var/log/pods` on a dedicated partition and at least XX GB

**Comment by nic-hima** [comment]
Would be interested in seeing a "best practices" doc for this sort of thing also. 

I'm needing to support RKE2 environments for years to come, so any efforts in advancing the documentation for it & it's differences from standard Rancher would be immense!
